File: ReadingApp/app/src/main/java/com/christianjcodes/readingapp/HomeActivity.java
package com.christianjcodes.readingapp;

import androidx.appcompat.app.AppCompatActivity;
import androidx.cardview.widget.CardView;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class HomeActivity extends AppCompatActivity {

    CardView dashboard;
    CardView currentReads;
    CardView completedReads;
    CardView readingList;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_home);

        dashboard = (CardView) findViewById(R.id.my_dashboard);
        currentReads = (CardView) findViewById(R.id.current_reads);
        completedReads = (CardView) findViewById(R.id.completed_reads);
        readingList = (CardView) findViewById(R.id.reading_list);

        dashboard.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(getApplicationContext(), DashActivity.class);
                startActivity(intent);
            }
        });

        currentReads.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(getApplicationContext(), CurrentReadsActivity.class);
                startActivity(intent);
            }
        });

    }
}

File: ReadingApp/app/src/main/java/com/christianjcodes/readingapp/HomeActivity.kt
package com.christianjcodes.readingapp

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import androidx.cardview.widget.CardView

class HomeActivity : AppCompatActivity() {
    var dashboard: CardView? = null
    var currentReads: CardView? = null
    var completedReads: CardView? = null
    var readingList: CardView? = null

    override fun onCreate(savedInstanceState: Bundle?) {

        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_home)
        dashboard = findViewById<View>(R.id.my_dashboard) as CardView
        currentReads = findViewById<View>(R.id.current_reads) as CardView
        completedReads = findViewById<View>(R.id.completed_reads) as CardView
        readingList = findViewById<View>(R.id.reading_list) as CardView

        dashboard!!.setOnClickListener {
            val intent = Intent(applicationContext, DashActivity::class.java)
            startActivity(intent)
        }
        currentReads!!.setOnClickListener {
            val intent = Intent(applicationContext, CurrentReadsActivity::class.java)
            startActivity(intent)
        }
        completedReads!!.setOnClickListener {
            val intent = Intent(applicationContext, CompletedReadsActivity::class.java)
            startActivity(intent)
        }
        readingList!!.setOnClickListener {
            val intent = Intent(applicationContext, ReadingListActivity::class.java)
            startActivity(intent)
        }
    }
}